site/content/docs/4.3/getting-started/theming.md

```diff
@@ -294,12 +294,18 @@ Sass cannot programmatically generate variables, so we must manually create them
 
 Our `tint-color()` and `shade-color()` functions use `mix()` alongside our `$theme-color-interval` variable, which specifies a stepped percentage value for each mixed color we need. See the `scss/_functions.scss` and `scss/_variables.scss` files for the full source code.
 
+{{< callout info >}}
+#### Monitoring contrast ratios
+
+Each theme color's swatch visually displays its current contrast ratio, as well as its contrast ratio against both white and black.
+{{< /callout >}}
+
```

Comment:
This could still be improved, but there's no need for a large callout here. If it's important, let's make it crisp and clear :).

```suggestion
Be sure to monitor contrast ratios as you customize colors. As shown below, we've added three contrast ratios to each of the main colors—one for the swatch's current colors, one for against white, and one for against black.
{{< /callout >}}

```

Reply:
Done :)